packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.ComponentPropsWithoutRef<'input'> {
```

Comment:
So we also missed the `multilineˋ prop that we had inside our internal library to show a t'exagéra conditionnaly?

Reply:
I'm on it 🚀